packages/builder/src/pages/builder/portal/apps/_FirstAppOnboarding/DatasourceConfigPanel.svelte

```diff
@@ -0,0 +1,108 @@
+<script>
+  import { Button, FancyForm, FancyInput, FancyCheckbox } from "@budibase/bbui"
+  import { capitalise } from "helpers/helpers"
+  import PanelHeader from "./PanelHeader.svelte"
+
+  export let title = ""
+  export let onBack = null
+  export let onNext = () => {}
+  export let fields = {}
+
+  let errors = {}
+
+  const formatName = name => {
```

Comment:
Function to format certain fields with irregular casing.

Reply:
Nice improvement actually over the actual data section!